integration-tests/tests/run-analysis-azure.js

```diff
@@ -0,0 +1,87 @@
+// This test is owned by the Interactive Analysis (IA) Team.
```

Comment:
This test should also be added to `slack-notify-channels.json`, which serves as a de facto list of test ownership.